core/invoker/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM scala
 
 # Uncomment to fetch latest version of docker instead: RUN wget -qO- https://get.docker.com | sh
-# Install docker 1.9.0
-RUN wget --no-verbose -O /usr/bin/docker "https://get.docker.com/builds/Linux/x86_64/docker-1.9.0" && \
+# Install docker 1.12.0 Client
+RUN wget --no-verbose https://get.docker.com/builds/Linux/x86_64/docker-1.12.0.tgz && \
+tar --strip-components 1 -xvzf docker-1.12.0.tgz -C /usr/bin docker/docker && \
+rm -f docker-1.12.0.tgz && \
```

Comment:
Well I suppose so. They just changed from single file to tar.gz but that was due to the split into multiple binaries. I think it should at least stay as tar.gz for the next couple of releases. I will update the PR accordingly.